src/views/htmlcontent/src/js/utils.ts

```diff
@@ -58,8 +58,9 @@ export function parseNumAsTimeString(value: number): string {
     let hs = h < 10 ? '0' + h : '' + h;
     let ms = m < 10 ? '0' + m : '' + m;
     let ss = s < 10 ? '0' + s : '' + s;
+    let mss = tempVal < 10 ? '00' + tempVal : tempVal < 100 ? '0' + tempVal : '' + tempVal;
```

Comment:
I guess I don't see the fundamental difference between how this is doing it and what you are suggesting. Only difference I see is if you are checking the length as a string or as a number.